src/NuProj.Tasks/GenerateNuSpec.cs

```diff
@@ -249,6 +265,21 @@ select new
                     }).ToList();
         }
 
+        private string GetContentFilesIncludePath(ITaskItem taskItem)
+        {
+            PackageDirectory packageDirectory;
+            string targetSubdirectory;
+            taskItem.GetTargetPackageDirectory(out packageDirectory, out targetSubdirectory);
+            if (packageDirectory != PackageDirectory.ContentFiles)
+            {
+                Log.LogError($"File '{taskItem.ItemSpec}' has unexpected PackageDirectory metadata. Expected '{PackageDirectory.ContentFiles}', actual '{packageDirectory}'.");
+
```

Comment:
Could I get you to clean up the empty lines that precede this and other of your closing curly brace lines?